openreview/journal/webfield/editorsInChiefWebfield.js

```diff
@@ -656,7 +687,7 @@ var renderData = function(venueStatusData) {
       papersAssigned: ['reviewerProgressData.numPapers'],
       default: ['summary.name']
     },
-    extraClasses: 'console-table'
+    extraClasses: 'eic-console-table'
```

Comment:
why did you have to change this class name?

Reply:
so that the column width can be overwritten

Reply:
moved to element
https://github.com/openreview/openreview-py/pull/1132/commits/04681f29ff7a8ae7dc93338006d4a6cb2e9d941c

Reply:
yes, in the post render table